app/src/main/res/values/strings.xml

```diff
@@ -81,17 +81,28 @@
     <string name="toast_dark_theme_off">Dark theme is now turned off.</string>
     <string name="contact_developer_on_telegram">Contact the developer on Telegram</string>
     <string name="txt_experimental_features">Experimental features</string>
-    <string name="toast_experimental_features_on">Experimental features turned on\nNew features could be unstable and unsafe</string>
-    <string name="toast_experimental_featured_off">Experimental features turned off</string>
+    <string name="toast_experimental_features_on">Experimental features turned on.\nNew features could be unstable and unsafe.</string>
+    <string name="toast_experimental_featured_off">Experimental features turned off.</string>
     <string name="txt_log_on_file">Save logs to a file</string>
     <string name="toast_log_on_file_on">Saving logs to a file turned on.\nYou can attach the log.txt file when you report an issue.\nRead more on GitHub.</string>
     <string name="toast_log_on_file_off">Saving logs to a file turned off.</string>
     <string name="txt_anonymous_statistics">Anonymous statistics</string>
     <string name="toast_anonymous_statistics_on">Anonymous statistics turned on.\nWe don\'t collect your personal data. We only use statistics to see how many people use the app each day.</string>
-    <string name="toast_anonymous_statistics_off">Anonymous statistics turned off.</string>
+    <string name="toast_anonymous_statistics_off">Anonymous statistics turned off.\nWe advice to reactivate them because we don\'t collect your personal data. We only use statistics to see how many people use the app each day.</string>
     <string name="txt_skip_recording_confirmation">Skip recording confirmation</string>
-    <string name="toast_skip_recording_confirmation_on">Skip recording confirmation turned on. You will now be able to send recordings without having to listen to them first.</string>
+    <string name="toast_skip_recording_confirmation_on">Skip recording confirmation turned on.\nYou will now be able to send recordings without having to listen to them first.</string>
     <string name="toast_skip_recording_confirmation_off">Skip recording confirmation turned off.</string>
+    <string name="txt_translate_the_app_on_crowdin">Translate the app on Crowdin</string>
+    <string name="txt_see_statistics_of_the_app">See official app statistics</string>
+    <string name="txt_recording_indicator_sound">Recording indicator sound</string>
+    <string name="toast_recording_indicator_sound_on">Recording indicator sound is now turned on.\nThe app will reproduce a sound when you start and stop to record a sentence.</string>
```

Comment:
"A sound will play when you start and stop recording."

Reply:
Sorry, I can't understand the correct line. Could you, please, specify it for every correction? Thanks a lot